src/Comments.jsx

```diff
@@ -330,6 +332,7 @@ Comments.propTypes = {
 Comments.defaultProps = {
     locale: 'eng',
     showVisibilityLevels: true,
+    disableAutofocus: false,
```

Comment:
Since we have defaultProps defined, maybe it would be worth to actually call the field simply `autofocus` and default it to true? What is the value of having negated fields exposed ?

Reply:
Sure, that makes sense. I will change it.